docs/backups.rst

```diff
@@ -1,25 +1,140 @@
-Backups
-=======
+CTS backups
+===========
 
-How backups are performed
--------------------------
+Backups are taken regularly and stored on the Caktus backup server. This document explains how to
+access and use those backups.
 
-The databases are hosted on AWS RDS and backups are run there.
+Getting a backup dump and restoring it locally
+----------------------------------------------
 
-How to check that backups are working
--------------------------------------
+Steps you can do ahead of time:
 
-Use the AWS Console and check the RDS instance's backups.  (TBD: where
-to look for that in the console.)
+* Get access to the Caktus backup server (open a tech support request).
 
-How to recover in the event of a lost server
+When you need to restore a backup:
+
+* Make sure you are in your CTS-IRCDeploy directory, and not in the CTS project directory::
+
+    $ git config --get remote.origin.url
+    git@github.com:theirc/CTS-ircdeploy.git
+
+* List the files in the latest backup directory and find the most recent::
+
+    $ backup_path=/mnt/rsnapshot/cts/daily.0/home/caktus-backup
+    $ ssh caktus-backup@backup.caktus.lan ls $backup_path | tail -1
+    cts_turkey.rescue.org-20170927.bz2
+
+  That will list the latest 'turkey' dumpfile. We'll replace 'turkey' with 'iraq' and 'jordan' to
+  get a dumpfile for each of the 3 instances.
+
+* Copy those files your local directory::
+
+    $ scp caktus-backup@backup.caktus.lan:${backup_path}/cts_iraq.rescue.org-20170927.bz2 cts_iraq.bz2
+    $ scp caktus-backup@backup.caktus.lan:${backup_path}/cts_jordan.rescue.org-20170927.bz2 cts_jordan.bz2
+    $ scp caktus-backup@backup.caktus.lan:${backup_path}/cts_turkey.rescue.org-20170927.bz2 cts_turkey.bz2
+
+  The iraq file is about 3MB, and the others are about 5MB each, as of Sept 2017.
+
+* Decompress the file using the `-k` flag which keeps the compressed version around (since we'll be
+  SCP'ing that to staging in a few steps)::
+
+    $ bunzip2 -k cts_iraq.bz2
+    $ bunzip2 -k cts_jordan.bz2
+    $ bunzip2 -k cts_turkey.bz2
+
+* Drop your existing local database and restore from the backup::
+
+    $ dropdb cts
+    $ createdb --template=template0 cts
+    $ psql cts -f cts_iraq > sql-import.log 2>&1
+
+  You can look through `sql-import.log` to view the output from that command. There will be a bunch
```

Comment:
This is a good idea, redirecting to a file so we can easily scan the output for real errors.

It looks like adding ``--quiet`` to the psql command cuts down some on the non-error output that clutters things up?